pkg/types/apisix/v1/types.go

```diff
@@ -463,3 +463,23 @@ func ComposeConsumerName(namespace, name string) string {
 
 	return buf.String()
 }
+
+// Schema represents the schema of APISIX objects.
+type Schema struct {
```

Comment:
If you changed the types.go, Please use the `make codegen` to genrate code.
`

Reply:
I did it but nothing updates.